src/FluentHub.App/FluentHub.App.csproj

```diff
@@ -118,21 +118,28 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Humanizer" Version="2.14.1" />
+    <PackageReference Include="Microsoft.AppCenter.Analytics" Version="4.5.3" />
+    <PackageReference Include="Microsoft.AppCenter.Crashes" Version="4.5.3" />
```

Comment:
@onein528 Can you add me to the AppCenter app group?

Reply:
Wdym?

Reply:
Looks like you added AppCenter analytics to FluentHub (obviously relies on an AppCenter account and registered app), so I'm asking if, via the AppCenter settings, you could add me to the app.
![image](https://user-images.githubusercontent.com/71598437/196042208-5ee1f900-3bea-4840-be16-465f0f033e4d.png)

Reply:
Definetely I will. But I removed AppCenter libraries(coz I added by mistake) and I will re-add them someday.